src/less-style/inputs.less

```diff
@@ -76,3 +76,22 @@
         width: 100%;
     }
 }
+
+.fd-commander {
+    display: flex;
```

Comment:
Is this still definitely necessary? Not blocking, but it'd be nice not to be using flex unless we confirm that our supported browsers all support flex (which I don't know off the top of my head).